README.md

```diff
@@ -230,3 +230,40 @@ var behavior = {};
 var stats = fetchBuilder(behavior).stats;
 console.log("Hit ratio", stats().cacheHitRatio);
 ```
+
+
+## Timeout
+Example:
+If you know the server response time is 3 seconds or 3000 ms you can configure exp-fetch timeout the following way.
+```
+timeout: {
+    socket: 3500,
+    request: 4000,   
+}
+```
+NOTE: This is a rare case. If have absolute control over responding server and have access to modify the timeout. See examples/timeout.js (to run copy the file to root and run with node). 
```

Comment:
If you have control over the responding server and have access to modify the timeout.